pkg/server/endpoints/endpoints_test.go

```diff
@@ -213,6 +217,7 @@ func TestListenAndServe(t *testing.T) {
 		Metrics:                      metrics,
 		RateLimit:                    rateLimit,
 		EntryFetcherCacheRebuildTask: ef.RunRebuildCacheTask,
+		EntryFetcherPruneEventsTask:  ef.PruneEventsTask,
```

Comment:
if we are adding default at this level, I think we must validate that value using a test